chsdi/templates/luftbilder/viewer.mako

```diff
@@ -134,9 +134,6 @@
     </style>
     <link rel="shortcut icon" type="image/x-icon" href="${h.versioned(request.static_url('chsdi:static/images/favicon.ico'))}">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
```

Comment:
Do you think we can remove this css ? 
We use only the bt style. We could copy the style we need directly in th HTML. It think it's good to avoid depending on external lib for this case.